stdlib/ballerina-builtin/src/main/ballerina/auth.authz.permissionstore/file-based-permission-store.bal

```diff
@@ -47,6 +51,22 @@ public function <FileBasedPermissionStore permissionStore> isAuthorized (string
     return matchGroups(groupsForScope, groupsForUser);
 }
 
+@Description {value:"Appends the contents of the second array to the first"}
+@Param {value:"primaryArray: name of the primary array to which content is appended"}
+@Param {value:"combiningArray: secondary array, elements of which will be appended to first"}
+@Return {value:"string[]: combined array"}
+function combineArrays (string[] primaryArray, string[] combiningArray) returns (string[]) {
+    if (lengthof combiningArray == 0) {
+        return primaryArray;
+    }
+    int i = 0;
+    while (i < lengthof combiningArray) {
+        primaryArray[lengthof primaryArray + i] = combiningArray[i];
+        i = i + 1;
```

Comment:
May be you can use increment statement Eg:- i++; :)

Reply:
Did not know ++ is supported now. Will use in future references. The reason to use the for loop was to get the index value.